docs/listfilepages/globalfilestagpages/globalfilesprexxx.md

```diff
@@ -0,0 +1,2236 @@
+
+<span id="prespellschool"></span> \*\*\* NEW 5.12
+
+**Tag Name:** PRESPELLSCHOOL:x,y=z,y=z
+
+**Variables Used (x):** Number (Number of spells known)
+
+**Variables Used (y):** Text (Name of a school of magic).
+
+**Variables Used (z):** Number (Minimum spell level)
+
+**What it does:**
+
+-   Establishes a requirement for (x) number of spells from magic school (y) of at least level (z).
+-   You may include as many "School=Level" pairs as you require as a comma-delimited (",") list.
+
+**Example:**
+
+`     PRESPELLSCHOOL:3,Necromancy=2    `
+
+Character must have at least three 2nd level or higher Necromancy spells to meet the requirement.
+
+------------------------------------------------------------------------
+
+<span id="prespellschoolsub"></span> \*\*\* Updated 5.12
+
+**Tag Name:** PRESPELLSCHOOLSUB:x,y=z,y=z
+
+**Variables Used (x):** Number (Number of spells known)
+
+**Variables Used (y):** Text (Name of sub-school of magic.)
+
+**Variables Used (z):** Number (Minimum spell level)
+
+**What it does:**
+
+-   Establishes a requirement for (x) number of spells from magic sub-school (y) of at least level (z).
+-   You may include as many "Sub-School=Level" pairs as you require as a comma-delimited (",") list.
+
+**Example:**
+
+`     PRESPELLSCHOOLSUB:3,Creation=2    `
+
+Character must have at least three 2nd level "Creation" spells to meet the requirement.
+
+------------------------------------------------------------------------
+
+<span id="prespelltype"></span> \*\*\* Updated 5.10
+
+**Tag Name:** PRESPELLTYPE:x,y=z
+
+**Variables Used (x):** Number (Number of spells known).
+
+**Variables Used (y):** Arcane (Arcane type spells).
+
+**Variables Used (y):** Divine (Divine type spells).
+
+**Variables Used (y):** Psionic (Psionic type spells).
+
+**Variables Used (y):** ANY (Any type of spell).
+
+**Variables Used (z):** Number (Spell level).
+
+**What it does:**
+
+-   Sets spell type requirements. (Number of spells known of a specific type and level.)
+-   Use two, or more, PRESPELLTYPE tags if more than one spell type is required
+
+**Example:**
+
+`     PRESPELLTYPE:4,Arcane=5    `
+
+Character must have at least four 5th level arcane spells to meet the requirement.
+
+`     PRESPELLTYPE:1,Arcane=3,Divine=3,Psionic=3    `
+
+Character must have at least one 3rd level spell from any of the "Arcane", "Divine", or "Psionic" types.
+
+`     PRESPELLTYPE:1,Arcane=3 <tab> PRESPELLTYPE:1,Divine=3    `
+
+Character must have at least one 3rd level "Arcane" type spell and one 3rd level "Divine" type spell.
+
+------------------------------------------------------------------------
+
+### PRESR
+
+Makes a character's spell resistance (not including SR from equipment) a prerequisite.
+
+#### Syntax
+
+`     PRESRoperator:spell_resistance    `
+
+`     operator    ` is a comparison operator from the list `     EQ    ` , `     LT    ` , `     LTEQ    ` , `     GT    ` , `    
```

Comment:
"base attack bonu and the epic attack bonus." should be "base attack bonus and the epic attack bonus."